generic3g/actions/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ target_sources(MAPL.generic3g PRIVATE
 
   TimeInterpolateAction.F90
   AccumulatorAction.F90
-  MeanAccumulator.F90
-  MaxAccumulator.F90
-  MinAccumulator.F90
+  MeanAction.F90
+  MaxAction.F90
+  MinAction.F90
```

Comment:
These may hold for now, but am concerned that they do not properly indicate that the mean/max/min are over time rather than space.   Hopefully clear in context, and invisible to external users in any event.